Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2336,14 +2292,31 @@ def to_context(self):
             if self.positive_detections:
                 domain_context['PositiveDetections'] = self.positive_detections
 
-            if self.whois:
-                domain_context['WHOIS'] = self.whois.to_context()
+            if self.registrar_name or self.registrar_abuse_email or self.registrar_abuse_phone:
+                domain_context['Registrar'] = {
```

Comment:
According to https://xsoar.pan.dev/docs/integrations/context-standards#domain  , Domain.Registrar doesn't exist. Only Domain.WHOIS.Registrar exist.